examples/00-fluent/mixing_elbow_settings_api.py

```diff
@@ -180,7 +180,10 @@
 # Solve for 150 iterations.
 
 solver.solution.run_calculation.iterate.get_attr("arguments")
-solver.solution.run_calculation.iterate(number_of_iterations=150)
+if float(solver.get_fluent_version().split(".")[0]) >= 23.0:
```

Comment:
`session.get_fluent_version() >= "23.1.0"` would be sufficeint

Reply:
Thank you for the suggestion. Done.